### DISCUSSION TITLE
Frigate 14 Questions

### DISCUSSION THREAD

**Discussion by DustyArmstrong** [blakeblackshear/frigate#15209] [Ask A Question]
Just migrated to v14 and had some questions with the change in UI, if anyone can assist.

1. I used to be able to toggle between playback types (JSMPEG, MSE etc) on the UI, is this no longer possible? A couple of my cameras keep using JSMPEG - I had it set to this for my Home Assistant cards to reduce bandwidth and changing it there to Go2Rtc also changed it on the web UI - why? Are the integration and cards linked to the web UI?
2. Is there any way to set the "Automatic Live View", "First Weekday" in config so they remain as-set? Every time I login they change back.
3. Same as above but for "Show Reviewed"?

This is more interest in a feature. Is there any plan to re-implement the playback scrub bar? I can use the timeline but this is very imprecise. Skipping back 10 seconds is fine but also not precise, it often skips to a totally different clip, it makes pausing on a frame more difficult.

**Comment by NickM-27** [comment]
1. This is not customizable on the Frigate webUI, the UI will automatically choose which stream type to use based on available bandwidth, camera codec, etc. The integration setting has no affect on how the Frigate UI works
2. It is saved in the browser, so as long as cookies / local storage is allowed to be saved then it will work
3. same as above

> This is more interest in a feature. Is there any plan to re-implement the playback scrub bar? I can use the timeline but this is very imprecise. Skipping back 10 seconds is fine but also not precise, it often skips to a totally different clip, it makes pausing on a frame more difficult.

you can use up / down arrow keys to move back and forth by one second. Timeline zooming will be supported in a future update

**Comment by DustyArmstrong** [comment]
Thanks, that's a shame with playback as v13 used to have more functionality, appreciate the quick reply.
